### PR TITLE
Modify README.md to support Chinese 

### PR DESCRIPTION
**Pull request by kwxmxb**
<docs>
[README]: Modify README.md to support Chinese language display.
<Modify README.md to support Chinese language, convenient for Chinese users to browse and watch, the developers hope that
Chinese language interface can be labeled in the README introduction page.>

### CODE REVIEW

**Comment by Siumauricio** [comment]
@kwxmxb can you revert the changes from the original Readme? and just let the chinese version?

File: README zhCN .md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Dokploy 是一个免费的可自行托管的平台即服务（PaaS），它使用 Docker 和 Traefik 简化了应用程序和数据库的部署和管理。 Dokploy 旨在提高效率和安全性，允许您在任何 VPS 上部署应用程序。
+
+
+## 🌟 特点
+
+
+- **应用**:轻松部署任何类型的应用程序（Node.js、PHP、Python、Go、Ruby 等）。
+- **数据库**:创建和管理数据库，支持 MySQL、PostgreSQL、MongoDB、MariaDB、Redis 等。
+- **Docker 管理**: 轻松部署和管理 Docker 容器。
+- **集成特拉菲克**: 自动与特拉菲克集成，用于路由选择和负载平衡。
```
Comment (hehehai) [inline README zhCN .md]: 这个应该是 

Traefik 集成，Traefik 本身不需要翻译

**Comment by hehehai** [comment]
Please do not translate the `README.md` file, just add the file `README-zh.md`

@Siumauricio I think README needs, README multi-language links, contribution file links, contributor list, License

like this：https://github.com/jaywcjlove/awesome-mac/blob/5df4431ec9ca2dbfe6acd6826697b6e1eb901038/README.md

File: README zhCN .md (hunk 1 above)
Comment (OSNewbie) [reply README zhCN .md]: https://github.com/Dokploy/dokploy/pull/68#issuecomment-2105662534

**Comment by hehehai** [comment]
@kwxmxb @Siumauricio I think the text content is ok, but there are some problems with the formatting and it needs to be consistent with the English document

image

**Comment by hehehai** [comment]
@kwxmxb Submitting, please do not delete the README.md file

**Comment by Siumauricio** [comment]
You are right @hehehai

@kwxmxb can you please do the following:

1. Can you please revert the main `Readme.md`, currently the file is deleted.
2. Fix the requested changes by @hehehai that is missing some content or mispelled
3. Can you also add this, a link that points to the different `Readme`languages?
image

**Comment by Siumauricio** [comment]
> @Siumauricio May I ask, is the default md translation file based on the official English or my modified Chinese? I don't quite understand.


Yes, the English readme file is the default.

**Review by Siumauricio — APPROVED** [review]

**Comment by Siumauricio** [comment]
Thanks @kwxmxb

**Comment by hehehai** [comment]
@Siumauricio just discovered a problem, this PR did not merge the commit 🤯

**Comment by Siumauricio** [comment]
@kwxmxb what do you mean? I see the changes on canary branch

**Comment by kwxmxb** [comment]
Dokploy panel language, which languages are supported I would like to ask

**Pull request merged**